APIs/openEO/authentication.qmd

```diff
@@ -0,0 +1,198 @@
+---
+title: "Authentication in openEO"
+---
+
+While basic discovery of openEO collections and processes is possible without authentication, executing openEO workflows requires user authentication. This is necessary to manage user quotas, resources, and credit consumption effectively. User authentication in openEO is handled with the OpenID Connect protocol (often abbreviated as "OIDC").
+
+The openEO endpoint of the Copernicus Data Space Ecosystem is configured to use the identity provider service of the Copernicus Data Space Ecosystem. It is therefore recommended to complete the [Copernicus Data Space Ecosystem registration](../../Registration.qmd) before attempting to authenticate with openEO.
+
+
+
+
+## Getting Started: A Typical openEO Authentication Flow With The openEO Python Client Library {#sec-typical-authentication-flow}
```

Comment:
Doesn't the sub-title seem to be a bit long?

Reply:
made it bit shorter